src/tools/linkchecker/main.rs

```diff
@@ -83,6 +83,7 @@ const LINKCHECK_EXCEPTIONS: &[(&str, &[&str])] = &[
     // This is fine though, because that section is hidden by the build system with `display: none`
     // when the link is missing. We thus ignore it to avoid a linkchecker complaint.
     ("index.html", &["qualification/technical-report.pdf"]),
+    ("release-notes/25.05.0.html", &["#check-cfg"]),
```

Comment:
worth adding comment of where that `#check-cfg` is from... I suspect it's from the inlined release notes